resources/cdk/python_example_code_apigateway_aws_service/setup.ts

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Purpose
+//
+// Defines a CloudFormation stack that creates an Amazon DynamoDB table with a
```

Comment:
"CloudFormation" -> "AWS CloudFormation" on first mention in a code file.

Reply:
Fixed.